frontend/src/routes/Credentials/CredentialsPage.tsx

```diff
@@ -231,13 +231,18 @@ export function CredentialsTable(props: {
             sort: 'metadata.creationTimestamp',
             cell: (resource) => (
               <span style={{ whiteSpace: 'nowrap' }}>
-                {resource.metadata.creationTimestamp && moment(new Date(resource.metadata.creationTimestamp)).fromNow()}
+                {resource.metadata?.creationTimestamp ? (
+                  <AcmTimestamp timestamp={resource.metadata.creationTimestamp} relative={true} />
+                ) : (
+                  '—'
+                )}
               </span>
             ),
             exportContent: (item: Secret) => {
               if (item.metadata.creationTimestamp) {
-                return moment(new Date(item.metadata.creationTimestamp)).fromNow()
+                return new Date(item.metadata.creationTimestamp).toISOString()
```

Comment:
I am not sure if we should make this change. There definitely is a problem here - we should not be exporting relative date-time.

When reviewing the fix for [ACM-16979](https://issues.redhat.com/browse/ACM-16979) by @Randy424, I was thinking about suggesting we use ISO 8601, which is the true standard for dates. However, I found that spreadsheet programs don't seem to like this format... Numbers at least will only recognize it as text. This is also true of the type of date-times that we are exporting where we have fixed it.

I suggest we leave it to @Randy424 to address this via some open defects he currently has to improve testing around exported values. We should export date-times in a consistent way, and it would be great if we can remove the use of `moment` altogether.